src/index.html

```diff
@@ -0,0 +1,1481 @@
+                    <!-- <li>
+                        <a href="#" class="text-sm xl:text-base  text-gray-600 hover:text-gray-900 uppercase">আত্মকর্ম </a>
+                    </li> -->
+                    <!-- link-৭ -->
+                    <li>
+                        <a href="#" class="text-sm xl:text-base  text-gray-600 hover:text-gray-900 uppercase">সামাজিক কার্জকম </a>
+                    </li>
+                    <!-- link-৮ -->
+                    <li class="flex items-center space-x-1 group relative">
+                        <a href="#" class="text-sm xl:text-base  text-gray-600 hover:text-gray-900 uppercase">গ্রন্থগার</a>
+                        <div class="flex items-center cursor-pointer">
+                            <svg class="w-4 h-4 group-hover:text-gray-900 text-gray-600" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor">
+                                <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
+                            </svg>
+                        </div>
+                        <div class=" bg-gray-100 h-40 absolute  top-10  w-40 z-10  opacity-0 group-hover:opacity-100 transition-all duration-500 invisible group-hover:visible">
+                            <div class="px-5 flex space-x-72 mt-5 ">
+                                <!-- === ড্রপডাউন ফ্লেক্সবক্স লেফট সাইট ==== -->
+                                <div class="flex flex-col">
+                                    <a href="#" class="text-sm  text-gray-600 hover:text-gray-800 uppercase ">গ্রন্থ তালিকা</a>
+                                    <a href="#" class="text-sm  text-gray-600 hover:text-gray-800 uppercase mt-4">গ্রন্থ বিপণন</a>
+                                    <a href="#" class="text-sm  text-gray-600 hover:text-gray-800 uppercase mt-4">সংগ্রহশালা</a>
+                                    <a href="#" class="text-sm  text-gray-600 hover:text-gray-800 uppercase mt-4">গবেষণা</a>
+                                </div>
+                            </div>
+                            <!-- === ড্রপডাউন ফ্লেক্সবক্স শেষ  ==== -->
+                        </div>
+                    </li>
+                    <!-- link-৯ -->
+                    <li>
+                        <a href="#" class="text-sm xl:text-base  text-gray-600 hover:text-gray-900 uppercase">রেস্তরা</a>
+                    </li>
+                    <!-- link-১০ -->
+                    <li>
+                        <a href="#" class="text-sm xl:text-base  text-gray-600 hover:text-gray-900 uppercase">গ্যালারী</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- header-links-last-section -->
+            <div>
+                <ul class="flex items-center space-x-2">
+                    <!-- link-1 -->
+                    <!-- link-2 -->
+                    <li class="flex items-center space-x-1 group hidden lg:hidden xll:flex">
```

Comment:
- `xll:flex` should be `xl:flex`
- There should be a `overflow: hidden` for mobile devices in responsive for the navigation item